### ISSUE TITLE
empty notification emails

### ISSUE THREAD

**Issue by mokkin** [nextcloud/activity#1521]
Usually the notification mails of activities have some content what has happened for example. Sind a couple of days I'm getting empty ones like this and I don't get what it wants to tell me:

![grafik](https://github.com/nextcloud/activity/assets/2938748/3b7d386d-72ab-439a-ab9e-cf02c4c40291)

**Operating system**:
Ubuntu 20.04
**Web server:**
Apache
**Database:**
MariaDB
**PHP version:**
8.2
**Nextcloud version:** (see Nextcloud admin page)
28.0.1
**Where did you install Nextcloud from:**
Owncloud upgrade many years ago

```
Login as admin user into your Nextcloud and access
http://example.com/index.php/settings/integrity/failed
paste the results here.

```
No errors have been found.

**Comment by mstoss** [comment]
Hello, I have the same issue with Nextcloud 27.1.5, php 8.1, debian 12. Many empty notifications are sent empty.

Where did you install Nextcloud from:
Owncloud upgrade many years ago

Database: mariadb

Web server: nginx

in the log:
`1","data":{"app":"activity"}}
{"reqId":"JuwODwZVLa7DXTOpnlfh","level":0,"time":"2024-01-20T21:00:06+00:00","remoteAddr":"","user":"--","app":"user_ldap","method":"","url":"--","message":"Calling LDAP function ldap_read with parameters [{},\"uid=gxxxx,ou=people,dc=inha,dc=fr\",\"objectClass=*\",[\"displayname\"],0,-1]","userAgent":"--","version":"27.1.5.1","data":{"app":"user_ldap"}}
{"reqId":"JuwODwZVLa7DXTOpnlfh","level":0,"time":"2024-01-20T21:00:06+00:00","remoteAddr":"","user":"--","app":"user_ldap","method":"","url":"--","message":"Calling LDAP function ldap_first_entry with parameters [{},{}]","userAgent":"--","version":"27.1.5.1","data":{"app":"user_ldap"}}
{"reqId":"JuwODwZVLa7DXTOpnlfh","level":0,"time":"2024-01-20T21:00:06+00:00","remoteAddr":"","user":"--","app":"user_ldap","method":"","url":"--","message":"Calling LDAP function ldap_get_attributes with parameters [{},{}]","userAgent":"--","version":"27.1.5.1","data":{"app":"user_ldap"}}
{"reqId":"JuwODwZVLa7DXTOpnlfh","level":0,"time":"2024-01-20T21:00:06+00:00","remoteAddr":"","user":"--","app":"no app in context","method":"","url":"--","message":"Email transport \"Symfony\\Component\\Mailer\\Transport\\Smtp\\SmtpTransport\" starting","userAgent":"--","version":"27.1.5.1","data":[]}
{"reqId":"JuwODwZVLa7DXTOpnlfh","level":0,"time":"2024-01-20T21:00:06+00:00","remoteAddr":"","user":"--","app":"no app in context","method":"","url":"--","message":"Email transport \"Symfony\\Component\\Mailer\\Transport\\Smtp\\SmtpTransport\" started","userAgent":"--","version":"27.1.5.1","data":[]}
{"reqId":"JuwODwZVLa7DXTOpnlfh","level":0,"time":"2024-01-20T21:00:06+00:00","remoteAddr":"","user":"--","app":"core","method":"","url":"--","message":"Sent mail to \"Array\n(\n    [gxxxx.pxxxx@inha.fr] => GXXXX PXXXXX\n)\n\" with subject \"Activit\u00e9 sur INHA Cloud\"","userAgent":"--","version":"27.1.5.1","data":{"app":"core"}}
{"reqId":"JuwODwZVLa7DXTOpnlfh","level":0,"time":"2024-01-20T21:00:06+00:00","remoteAddr":"","user":"--","app":"no app in context","method":"","url":"--","message":"Email transport \"Symfony\\Component\\Mailer\\Transport\\Smtp\\SmtpTransport\" stopping","userAgent":"--","version":"27.1.5.1","data":[]}
{"reqId":"JuwODwZVLa7DXTOpnlfh","level":0,"time":"2024-01-20T21:00:06+00:00","remoteAddr":"","user":"--","app":"no app in context","method":"","url":"--","message":"Email transport \"Symfony\\Component\\Mailer\\Transport\\Smtp\\SmtpTransport\" stopped","userAgent":"--","version":"27.1.5.1","data":[]}
{"reqId":"JuwODwZVLa7DXTOpnlfh","level":0,"time":"2024-01-20T21:00:06+00:00","remoteAddr":"","user":"--","app":"activity","method":"","url":"--","message":"Activity \"\" was not parsed by any provider","userAgent":"--","version":"27.1.5.1","data":{"app":"activity"}}`

**Comment by kunago** [comment]
I recently upgraded to NC28 and having the same issue. Hard to tell whether it's an app issue. I looked at the database, the table "oc_activity_mq" and it is properly populated with changes that should be sent later on. But instead of being sent, an empty template is sent. So is it likely to be a template issue?

**Comment by kunago** [comment]
It might be linked with deck but I am unable to verify as I upgraded a few versions at once. I am on deck 1.12.1. What versions are you running? And can you think of a version which worked, unless you upgraded NC as well?

**Comment by mstoss** [comment]
I'm using 1.11.3 with nextcloud 27.1.5. I think 1.11.3 is the last compatible version with NC 27.  But both versions were released last week.

**Comment by mstoss** [comment]
Still the problem in 27.1.6

**Comment by susnux** [comment]
yes this happens due to invalid rich objects set by deck app

**Comment by kunago** [comment]
I can on the other hand confirm the Deck notifications are passing fine. The issue has been fixed on my installation, running NC 28.0.2 and Deck 1.12.2.

**Comment by susnux** [comment]
This is fixed in Deck 1.12.2 

Please update the deck app on your instance, if it still happens after the update, please reopen.

**Comment by MichaelPerlbach** [comment]
Still no email notifications with Deck 1.12.2 in Nextcloud 28.0.3.
